testsuite/features/step_definitions/command_steps.rb

```diff
@@ -180,7 +180,11 @@
 
 When(/^I query latest Salt changes on "(.*?)"$/) do |host|
   node = get_target(host)
-  result, return_code = node.run("LANG=en_US.UTF-8 rpm -q --changelog salt")
+  salt = 'salt'
+  if host != 'server' && host != 'proxy'
+    salt = $product == 'Uyuni' ? "venv-salt-minion" : "salt"
+  end
+  result, return_code = node.run("LANG=en_US.UTF-8 rpm -q --changelog #{salt}")
```

Comment:
@mcalmer maybe just modify the command this way?
```
rpm -q --changelog venv-salt-minion || rpm -q --changelog salt
```

Reply:
I prefer Michael's more deterministic approach.

Reply:
As we are distinguished by product version in the rest of the steps... better do it also here.

Reply:
ok